src/renderer/components/table/table-cell.tsx

```diff
@@ -20,9 +24,98 @@ export interface TableCellProps extends React.DOMAttributes<HTMLDivElement> {
   _sorting?: Partial<TableSortParams>; // <Table> sorting state, don't use this prop outside (!)
   _sort?(sortBy: TableSortBy): void; // <Table> sort function, don't use this prop outside (!)
   _nowrap?: boolean; // indicator, might come from parent <TableHead>, don't use this prop outside (!)
+
+  /**
+   * Triggers every time this cell is resized by user
+   * @param width desired width of this cell
+   */
+  _onResize?(width: number): void;
```

Comment:
Why are these underscore prefixed, that visually seems to imply that they are some sort of "private" or for "internal testing only"

Reply:
After multiple refactors of this part I can no longer recall my motivation for this kind of naming tbh, changing it back to normal.